pydis_site/apps/content/resources/guides/python-guides/discordpy_help_command.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Custom Help Command
+description: "Overwrite discord.py's help command to implement custom functionality"
+---
+
+# Custom Help Command
+
```

Comment:
We won't need the `# Custom Help Command`, as it's already in the title, so it'll repeat as seen below:
![image](https://user-images.githubusercontent.com/82843611/159189850-ce272c7a-02f5-4eb4-98bd-7aa740879593.png)
```suggestion
```